src/Services/QueuedJobService.php

```diff
@@ -603,10 +607,12 @@ public function checkDefaultJobs($queue = null)
                             );
                             continue;
                         }
-                        QueuedJobService::singleton()->queueJob(
-                            Injector::inst()->createWithArgs($jobConfig['type'], $jobConfig['construct']),
-                            date($jobConfig['startDateFormat'] ?? '', strtotime($jobConfig['startTimeString'] ?? ''))
+                        $queuedJob = Injector::inst()->createWithArgs($jobConfig['type'], $jobConfig['construct']);
+                        $queuDate = date(
+                            $jobConfig['startDateFormat'] ?? '',
+                            strtotime($jobConfig['startTimeString'] ?? '')
                         );
+                        $this->queueJob($queuedJob, $queuDate);
```

Comment:
We shouldn't be doing this - we should be using the singleton which presumbably is being used as the central class to keep a record of all the data